src/daemon/database/schema.rs

```diff
@@ -46,6 +46,7 @@ CREATE TABLE vaults (
     wallet_id INTEGER NOT NULL,
     status INTEGER NOT NULL,
     blockheight INTEGER NOT NULL,
+    received_at INTEGER NOT NULL,
```

Comment:
Bikeshedding but just to cure my OCD i feel like it should be just before `updated_at`